index.html

```diff
@@ -3142,39 +3145,45 @@ <h3 id="docconformance-attr">
           </tbody>
         </table>
       </section>
-    </section>
-    <section>
+      <section>
       <h2 id="case-sensitivity">
         Case requirements for ARIA role, state and property attributes
       </h2>
       <p>
-        Authors MUST use <a data-cite=
+        Authors SHOULD use <a data-cite=
         "html/infrastructure.html#lowercase-ascii-letters">lowercase ASCII
         letters</a> for all `role` token values and any state or property
         attributes (`aria-*`) whose values are <a data-cite=
         "wai-aria-1.1#propcharacteristic_value">defined as tokens</a>.
       </p>
+
+      <div class="note">
+        <p>While all modern browsers allow for the use of upper or lowercase ASCII letters to specify the `role` or `aria-*` attribute state and property values, not all assistive technologies will correctly parse these values.</p>
```

Comment:
"upper or lowercase" > "upper- or lowercase"

Reply:
Any case? Case insensitive?

Reply:
```suggestion
        <p>While browsers treat the `role` or `aria-*` attribute values as [=ascii case-insensitive=], not all assistive technologies will correctly parse these values.</p>
```

Still feel pretty weak to give other assistive technology a pass here (they are non-conforming and we should fix them). I guess we can update this over time as things improve.

Reply:
it accepts the reality though. it's no good if authors do it "per spec", it doesn't actually work for people with disabilities / using AT, but authors shrug and say well not my fault.

"we should fix them" good luck with that, some assistive technologies are black boxes of mystery, with very unresponsive development teams. we've tried nudging them on other things in the past, often with no response/success.

Reply:
> "we should fix them" good luck with that, 

Fixing Firefox took a couple of minutes, so I'm hopeful. If you know of open source ones that are broken, I can try to fix them. "It's just computers"™️, after all. 

I accept that there is not much we can do about the closed source ones if they are unresponsive.